src/main/java/mate/academy/dao/impl/CinemaHallDaoImpl.java

```diff
@@ -9,16 +9,23 @@
 import mate.academy.model.CinemaHall;
 import mate.academy.util.HibernateUtil;
 import org.hibernate.Session;
+import org.hibernate.SessionFactory;
 import org.hibernate.Transaction;
 
 @Dao
 public class CinemaHallDaoImpl implements CinemaHallDao {
+    private final SessionFactory sessionFactory;
+
+    public CinemaHallDaoImpl() {
+        this.sessionFactory = HibernateUtil.getSessionFactory();
```

Comment:
this is redundant

Reply:
I disagree. I disagree. Referring to the HibernateUtil class all the time is redundant.

Reply:
ops. okey i got it.